<launch>
	<arg name="sim_viewer_arg" default="gazebo" /> <!-- gazebo or rviz or none-->

	<param name="ik" type="str" value="trac_ik" />

	<rosparam file="$(find robetarme)/config/param.yaml" />

	<group if="$(eval arg('sim_viewer_arg') == 'gazebo')">
		<include file="$(find iiwa_gazebo)/launch/iiwa_gazebo.launch">
			<arg name="controller" default="PositionController" />
			<arg name="model" default="7" />
		</include>
	</group>

    <node pkg="robetarme" type="path_planning_node.py" name="pathPlanningNode" output="screen"></node>
</launch>